source/DataFile.h

```diff
@@ -44,6 +45,8 @@ class DataFile {
 	std::list<DataNode>::const_iterator begin() const;
 	std::list<DataNode>::const_iterator end() const;
 
+	std::shared_ptr<ConditionsStore> Store();
```

Comment:
ik you're trying to make variable names short but I think it's best to have clarity and to keep it as ConditionStore (at least for the function, not necessarily for the data member; how about having that one named condStore if you want it short?)

Reply:
ConditionStore would be confusing since the type is Condition**s**Store. I can't make it ConditionsStore because of the resulting name clash. With those restrictions, "store" seems best, since there's only one type of store that provides conditions.

Reply:
we could name it CondStore but I guess its fine as Store too

Reply:
I think, given that you're working with a Condition, it is reasonable for the reader to assume the Condition's "store" is a ConditionsStore and an "element" is a ConditionElement.